### PR TITLE
Perf/point cloud seg

### PR DESCRIPTION
**Pull request by Kerwin-L**

### CODE REVIEW

File: packages/lb-annotation/src/core/pointCloud/index.ts [hunk 1]
```diff
@@ -838,21 +841,49 @@ export class PointCloud extends EventListener {
   public renderPointCloud(points: THREE.Points, radius?: number) {
     points.name = this.pointCloudObjectName;
 
-    const pointsMaterial = new THREE.PointsMaterial({
-      vertexColors: true,
-    });
+    const material = new THREE.ShaderMaterial({
```
Comment (Glenfiddish) [inline packages/lb-annotation/src/core/pointCloud/index.ts:844]: explain ShaderMaterial usage and refactor as a function?

File: packages/lb-annotation/src/core/pointCloud/index.ts [hunk 2]
```diff
@@ -874,11 +905,7 @@ export class PointCloud extends EventListener {
       return;
     }
     for (let i = 0; i < points.length; i += 3) {
-      const x = points[i];
-      const y = points[i + 1];
-      const z = points[i + 2];
-
-      this.store.cloudData.set(`${x}@${y}@${z}`, { visible: false });
+      this.store.cloudData.set(`${points[i]}@${points[i + 1]}@${points[i + 2]}`, { visible: false });
```
Comment (Glenfiddish) [inline packages/lb-annotation/src/core/pointCloud/index.ts:908]: is same?

File: packages/lb-annotation/src/core/pointCloud/index.ts (hunk 1 above)
Comment (MMMMoriaty) [inline packages/lb-annotation/src/core/pointCloud/index.ts:844]: what about the points in pointcloud renderer?

File: packages/lb-annotation/src/core/pointCloud/index.ts (hunk 2 above)
Comment (Kerwin-L) [reply packages/lb-annotation/src/core/pointCloud/index.ts:908]: Reduce the declaration

File: packages/lb-annotation/src/core/pointCloud/index.ts (hunk 1 above)
Comment (Kerwin-L) [reply packages/lb-annotation/src/core/pointCloud/index.ts:844]: No need, these two types of materials can coexist. The key of update is to filter the points by shader.

**Pull request merged**